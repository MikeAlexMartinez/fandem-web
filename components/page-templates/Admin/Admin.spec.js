import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import AdminComponent from './Admin';

describe('<Admin />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<AdminComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
