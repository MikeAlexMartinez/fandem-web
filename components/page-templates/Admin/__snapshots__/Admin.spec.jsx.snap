// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Admin /> renders and matches the snapshot 1`] = `
<WithStyles(Page)>
  <WithStyles(PleaseSignIn)>
    <WithStyles(Navigation)>
      <WithStyles(IsAdmin)>
        <WithStyles(UsersGeneralInfo) />
      </WithStyles(IsAdmin)>
    </WithStyles(Navigation)>
  </WithStyles(PleaseSignIn)>
</WithStyles(Page)>
`;
